schema.md

```diff
@@ -109,6 +109,29 @@
     \_ga=GA1.2.1076416180.1513383458; \_gid=GA1.2.1940452730.1513383458
     {"name": "example", "Browser": "Mozilla/5.0"}
     ```
+* __value_1000:__
+    * __Type:__ string
+    * __Description:__ The value that the function returned, restricted to 1000 characters.
```

Comment:
Nit: I realize this is being picky, but I think the word "truncated" is more appropriate than "restricted".